SparkPost/2web2ui
# SA-1984: use usernames for recipients in Scheduled Reports

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed

- Pass username into scheduled reports api instead of name.

### How To Test

- Create a report here `/signals/schedule/bounce`
- Add recipients with custom names. IE, for {name}<{email}>, their name does not follow the `first-last-####`.
- Verify that the API receives username, not name as the values for recipients.
- Also use `Send Test` button to verify that the email is being sent.

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
